Libs/MRML/Core/vtkMRMLSegmentationNode.cxx

```diff
@@ -460,8 +460,16 @@ void vtkMRMLSegmentationNode::GetRASBounds(double bounds[6])
     {
     this->Segmentation->GetBounds(bounds);
     }
+
 }
 
+//---------------------------------------------------------------------------
+void vtkMRMLSegmentationNode::GetNodeBounds(double bounds[6])
+{
+  Superclass::GetNodeBounds(bounds);
```

Comment:
In general, nodes return information without applying the parent transform, see for example Volume node's GetOrigin() or Markup fiducial node's GetNthFiducialPosition - they all provide position in the node's coordinate system, not in RAS. Having _RAS_ in the name of GetRASBounds indicates that it's an exception. Probably not the best name, but it's not too bad and it would be difficult to change, so for now I would not touch it.